Orange/widgets/__init__.py

```diff
@@ -17,6 +17,7 @@ def widget_discovery(discovery):
         "Orange.widgets.visualize",
         "Orange.widgets.classify",
         "Orange.widgets.regression",
+        "Orange.widgets.learners",
```

Comment:
@BlazZupan, will the merged category (classify + regression) be called "learners"?

- We have "Unsupervised", so to keep in line with that the new category would have to be "Supervised". It's weird, I don't like it.
- A better option is "Modelling". The form doesn't fit, but we already have a mess -- "Visualize", "Classify" and "Regression", and not "Visualization", "Classification" and "Regression" (or "Visualize", "Classify" and "Regress"!?).
- The problem with "Modelling" is that it also covers unsupervised modelling.

Still, I vote for "Model" or "Modelling".